### PR TITLE
Fix (Airdrops): `has_decoder` field

### PR DESCRIPTION
**Pull request by OjusWiZard**
## Checklist

- [x] Parse a `has_decoder` field from the airdrops' index and propagate it to the frontend if it's `false`.
- [x] Update the docs.

### CODE REVIEW

File: docs/contribute.rst [hunk 1]
```diff
@@ -392,6 +394,29 @@ This is how rotki inserts a variable inside a sentence. You **must** keep this v
...
+- If the token is new and is not expected to exist in the userDB and globalDB then ``new_asset_data`` should be added in the index.
+- If the asset's icon is not present in ``rotki`` repo, it should be added in ``airdrops/icons`` directory, with its path provided in its airdrop's index entry.
+- If no decoder is present to decode the claiming event of this airdrop, add a ``has_decoder`` field in the entry with the value as ``false``.
+
+Adding Token Airdrops with a CSV
+--------------------------------
+
+- Add the CSV in the ``airdrops/`` directory, with its path provided in its index entry.
+    - The CSV's header should have first column as ``address`` and second column as ``amount``.
+    - The amounts should be normalised according to their decimals.
```
Comment (yabirgb) [inline docs/contribute.rst:412]: there are extra tabs here

File: docs/contribute.rst [hunk 2]
```diff
@@ -392,6 +394,29 @@ This is how rotki inserts a variable inside a sentence. You **must** keep this v
...
 3. For missing keys from other language files, by default it will use the value of the master file which is ``English``.
 
+Add a new Airdrop
+===================================
+
+To add a new airdrop in the DeFi/Airdrops section, go to the `data repo here <https://github.com/rotki/data/tree/develop>`__.
+
+- Add the entry in ``airdrops/index_v2.json`` under the ``airdrops`` field.
+- If the token is new and is not expected to exist in the userDB and globalDB then ``new_asset_data`` should be added in the index.
+- If the asset's icon is not present in ``rotki`` repo, it should be added in ``airdrops/icons`` directory, with its path provided in its airdrop's index entry.
```
Comment (yabirgb) [inline docs/contribute.rst:404]: mention what to do if we have the icon

File: docs/contribute.rst (hunk 1 above)
Comment (OjusWiZard) [reply docs/contribute.rst:412]: those are the indented bullet points because they add more info to the first point

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**